virttest/qemu_monitor.py

```diff
@@ -968,7 +968,7 @@ def set_block_job_speed(self, device, speed=0,
         if correct:
             cmd = self.correct(cmd)
         self.verify_supported_cmd(cmd)
-        cmd += " %s %sB" % (device, speed)
+        cmd += " %s %sB" % (device.split(" ")[0], speed)
```

Comment:
@xutian Agree, 'device' here does not depends on qemu version, and we can get accurate device name by qemu_vm.get_block(), it works well on both qemu 2.6 and qemu 1.5. Why this modification?